runelite-client/src/main/java/net/runelite/client/plugins/skillcalculator/skills/SkillAction.java

```diff
@@ -96,4 +96,9 @@ default boolean isBonusApplicable(SkillBonus bonus)
 	}
 
 	boolean isMembers(final ItemManager itemManager);
+
+	default SkillBonus[] getExcludedSkillBonuses()
```

Comment:
Is there a reason for this not to be `Set<SkillBonus>`? That would avoid unnecessary Stream uses (can just use `Collection#contains()` calls instead) and is semantically more accurate. (can't exclude a skill bonus multiple times--whatever that would mean?) I'd be open to changing the shape of `PrayerBonus.BONE_BONUSES` if that's the reason this returns an array.